Source/WebCore/crypto/keys/CryptoKeyRSA.cpp

```diff
@@ -133,6 +133,7 @@ JsonWebKey CryptoKeyRSA::exportJwk() const
     JsonWebKey result;
     result.kty = "RSA"_s;
     result.key_ops = usages();
+    result.usages = usagesBitmap();
```

Comment:
Can this call a JsonWebKey constructor with members rather than calling the empty constructor then populating?

Reply:
I think it can, but `JsonWebKey` currently doesn't have constructors and I tried to use what's available to keep change size small.
May it do as followup.